<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,0" columnstretch="1,2,2,2,2">
    <item row="0" column="3">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Google Satelite</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Google Hybrid</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Google Terrain</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Google Streets</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue Marble</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>City Lights</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Open Street Map</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Plain</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SRTM</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="projection_bottun">
      <property name="text">
       <string>Spherical</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="coordinations_indicator_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QWidget" name="base_widget" native="true">
      <property name="styleSheet">
       <string notr="true">border-radius: 40px;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="coordinations_label">
      <property name="text">
       <string>mouse coords: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
